test_runner/regress/test_remote_storage.py

```diff
@@ -333,7 +336,7 @@ def churn_while_failpoints_active(result):
 
     # The churn thread doesn't make progress once it blocks on the first wait_completion() call,
     # so, give it some time to wrap up.
-    churn_while_failpoints_active_thread.join(30)
+    churn_while_failpoints_active_thread.join(60)
```

Comment:
I think we could go straight for 120, or just monitor how it'll go with this.

Reply:
the error in here seems to be more rare. let's see how we fare with 60 first.